### PR TITLE
Several Changes : 

### PR DESCRIPTION
**Pull request by naeloob**
Mediator : Fix to work on windows.
Mediator : Checks the ports to look for the first not open.
Fix words comand : avoid an http redirect that generates an error.
Fix move command : Notepad for windows as editor, fix cannot open a temporary open file, add encoding='utf-8'  to void char problems.
Add Focus command : It will activate the Tab and focus the window. It doesn't always bring to front, but it highlight it and helps to find it. Usefull if you have a lot of windows and need to know where is the tab.
Fix query : fixed an inconsistency in the number of parameters in the query method call
NEW: parameter to use a ',' separate list of IP:PORT to connect. If you have too much browsers to check, you can point unique to a list of them. It allows to point to a browser on another IP, not only localhost, without the need of network tweak(as port redirect).

Still testing, but with this patches i can make a great use on Windows/Chrome. Hope it will work ok 
 on the other OS/Browser combinations.
It will need more changes for Windows/FF.

### CODE REVIEW

File: brotab/inout.py [hunk 1]
```diff
@@ -9,35 +9,41 @@
...
+    if os.path.exists(filename):
+        os.remove(filename)
+    return Lines
 
 
 def edit_tabs_in_editor(tabs_before):
     with NamedTemporaryFile() as file_:
-        save_tabs_to_file(tabs_before, file_.name)
+        file_name=file_.name
+        file_.close()
```
Comment (balta2ar) [inline brotab/inout.py]: @naeloob why is this necessary?

Comment (naeloob) [reply brotab/inout.py]: Because NamedTemporaryFiler lefts the file opened and, almost in windows, when save_tabs_to_file try to open it, it fails. That way, we get the temporary filename and close the file.
I've seen an error en my line 41 : 
`check_call([os.environ.get('EDITOR', Editor), file_.name])`
must be : 
`check_call([os.environ.get('EDITOR', Editor), file_name])`

File: brotab/main.py [hunk 2]
```diff
@@ -398,6 +422,17 @@ def parse_args(args):
     parser_activate_tab.add_argument('tab_id', type=str, nargs=1,
                                      help='Tab ID to activate')
 
+    parser_activateFocus_tab = subparsers.add_parser(
```
Comment (balta2ar) [inline brotab/main.py]: @naeloob I don't think more command is necessary. Instead, I'd like to add a `--focus` flag to `activate` command.

Comment (balta2ar) [reply brotab/main.py]: @naeloob for the record, I'm adjusting your pull request in another pull request here: https://github.com/balta2ar/brotab/pull/22
I didn't want to bother you with the code review so I decided to fix the code according to my taste. I hope it's OK with you.

Comment (naeloob) [reply brotab/main.py]: > @naeloob I don't think more command is necessary. Instead, I'd like to add a `--focus` flag to `activate` command.

That's rigth. I didn't know how to add a flag :)

> @naeloob for the record, I'm adjusting your pull request in another pull request here: #22
> I didn't want to bother you with the code review so I decided to fix the code according to my taste. I hope it's OK with you.

No problem friend.

File: brotab/extension/background.js [hunk 3]
```diff
@@ -102,16 +113,58 @@ class FirefoxTabs extends BrowserTabs {
   getBrowserName() {
       return "firefox";
   }
+
+  activate(tab_id) {
+    this._browser.tabs.update(tab_id, {'active': true});
```
Comment (balta2ar) [inline brotab/extension/background.js]: @naeloob  The documentation says "focused" but you use "active". Is that a typo?

**Comment by balta2ar** [comment]
Thanks a lot for the pull request, I reworked it a bit and merged in #22.
